packages/vc-export/src/CredentialSchema.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2018-2023, BOTLabs GmbH.
+ *
+ * This source code is licensed under the BSD 4-Clause "Original" license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+import { CType } from '@kiltprotocol/core'
+import type { ICType } from '@kiltprotocol/types'
+import { jsonLdExpandCredentialSubject } from './common.js'
+import type { VerifiableCredential } from './types.js'
+
+/**
+ * Validates the claims in the VC's `credentialSubject` against a CType definition on the `credentialSchema` property.
+ *
+ * @param credential A verifiable credential where `credentialSchema.schema` is an [[ICType]].
+ * @param credential.credentialSubject The credentialSubject to be validated.
+ * @param credential.credentialSchema The credentialSchema to be applied.
+ * @param cType Optionally pass the CType definition to be used if it is not embedded in the credentialSchema.
+ */
+export function validateSchema(
```

Comment:
Because it's exported under its own namespace, would it make sense to simply rename it to `validate`?

Reply:
Maybe `validateSubject` because the schema is assumed not to validate the whole credential; it just checks the credentialSubject.

Reply:
fba870433efcd82c894e2a95cea764a4620e8bbc